python/lsst/ctrl/oods/msgIngester.py

```diff
@@ -162,6 +172,12 @@ def _gather_all_resource_paths(self, m):
         # extract all urls within this message
         msg = BucketMessage(m.value())
 
-        rp_list = [ResourcePath(url) for url in msg.extract_urls()]
+        names = [url for url in msg.extract_urls()]
+
+        name_list = self.filter_by_regex(names)
```

Comment:
It looks like `urls` and `filtered_urls` could be an improvement over generic `names` and `name_list` in terms of the code readability, but since the function is so simple I'm fine with leaving things as they are.

Reply:
Fixed